pkg/cmd/cli/client/config/config.go

```diff
@@ -18,20 +18,17 @@ package config
 
 import (
 	"github.com/spf13/cobra"
-
-	"github.com/heptio/ark/pkg/client"
 )
 
-func NewCommand(f client.Factory) *cobra.Command {
+func NewCommand() *cobra.Command {
 	c := &cobra.Command{
 		Use:   "config",
-		Short: "Work with config",
-		Long:  "Work with config",
+		Short: "Get and set client configuration file values",
```

Comment:
@Bradamant3 could use some word-smithing here. Is it worthwhile to have `Long` and `Example` as well?